RELEASES.md

```diff
@@ -1,3 +1,149 @@
+* Process management
+    * [Support `CLONE_PARENT` flag](https://github.com/asterinas/asterinas/pull/2447)
+    * [Support `execve` in multithreaded process](https://github.com/asterinas/asterinas/pull/2459)
+    * [Support `PR_SET`/`GET_SECUREBITS`](https://github.com/asterinas/asterinas/pull/2551)
+    * [Fix some `kill`-related behavior](https://github.com/asterinas/asterinas/pull/2516)
+* IPC
+    * [Add `rt_sigtimedwait` syscall](https://github.com/asterinas/asterinas/pull/2705)
+    * [Enqueue ignored signals if the signals are blocked](https://github.com/asterinas/asterinas/pull/2503)
+    * [Refactor `NamedPipe` to correct its opening and blocking behaviors](https://github.com/asterinas/asterinas/pull/2434)
+    * [Support reopening anonymous pipes from `/proc`](https://github.com/asterinas/asterinas/pull/2694)
+    * [Fix or clarify some futexes bugs](https://github.com/asterinas/asterinas/pull/2515)
+* File systems
+    * VFS
+        * [Add `inotify`-related syscalls](https://github.com/asterinas/asterinas/pull/2083)
+        * [Add new ioctl infrastructure](https://github.com/asterinas/asterinas/pull/2686)
+        * [Add `chmod` and `mkmod` macros](https://github.com/asterinas/asterinas/pull/2440)
+        * [Add `syncfs` syscall](https://github.com/asterinas/asterinas/pull/2682)
+        * [Add `fchmodat2` syscall](https://github.com/asterinas/asterinas/pull/2666)
+        * [Support mount bind with a file](https://github.com/asterinas/asterinas/pull/2418)
+        * [Support `MS_REMOUNT` flag](https://github.com/asterinas/asterinas/pull/2432)
+        * [Ensure that every `FileLike` is associated with a `dyn Inode`](https://github.com/asterinas/asterinas/pull/2555)
+    * Devtmpfs
+        * [Add `/dev/full` device](https://github.com/asterinas/asterinas/pull/2439)
+        * [Support registering char devices](https://github.com/asterinas/asterinas/pull/2598)
+        * [Support registering block device (and their partitions)](https://github.com/asterinas/asterinas/pull/2560)
+    * Procfs
+        * [Add `/proc/cmdline` support](https://github.com/asterinas/asterinas/pull/2420)
+        * [Add `/proc/stat` and `/proc/uptime`](https://github.com/asterinas/asterinas/pull/2370)
+        * [Add `/proc/version`](https://github.com/asterinas/asterinas/pull/2679)
+        * [Add `/proc/[pid]/environ`](https://github.com/asterinas/asterinas/pull/2371)
+        * [Add `/proc/[pid]/oom_score_adj`](https://github.com/asterinas/asterinas/pull/2410)
+        * [Add `/proc/[pid]/cmdline` and `/proc/[pid]/mem`](https://github.com/asterinas/asterinas/pull/2449)
+        * [Add `/proc/[pid]/mountinfo`](https://github.com/asterinas/asterinas/pull/2399)
+        * [Add `/proc/[pid]/fdinfo`](https://github.com/asterinas/asterinas/pull/2526)
+        * [Add `/proc/[pid]/maps`](https://github.com/asterinas/asterinas/pull/2725)
+        * [Support the sleeping and stopping states in `/proc/[pid]/stat`](https://github.com/asterinas/asterinas/pull/2491)
```

Comment:
- This list seems a bit longer. If there's room for one more, I suggest adding https://github.com/asterinas/asterinas/pull/2553.